src/helper.rs

```diff
@@ -287,6 +287,9 @@ pub fn parse_timestamp(value: &str) -> Result<OffsetDateTime> {
     let utc_format = format_description::parse(
         "[year][month][day]T[hour][minute][second]Z",
     )?;
+    let rfc3339_format = format_description::parse(
+        "[year]-[month]-[day]T[hour]:[minute]:[second]Z",
+    )?;
```

Comment:
Can we use `format_description::well_known::Rfc3339` here instead please?

Reply:
Hmm, this is a slightly different format. It also includes a fraction of a second after the last `.`:

```
1985-04-12T23:20:50.52Z
```

So, I guess, I also incorrectly named it RFC3339 here :)

I've been trying to parse my vcards with this fix, and there are still a lot of problems with them (not with the library, but because the cards violate the spec).

So I think I'll use `parse_loose` instead, fix the incorrect fields and regenerate the cards.

Reply:
So, for the sake of keeping the library RFC-correct, I suggest to close the PR.

Reply:
Thanks for the info, closing as suggested.